src/vanilla/GeneratorSettingsJs.cs

```diff
@@ -11,17 +11,22 @@ public class GeneratorSettingsJs : IGeneratorSettings
         /// <summary>
         /// Whether or not to generate a new package.json file.
         /// </summary>
-        public bool GeneratePackageJson { get; set; }
+        public bool GeneratePackageJson { get; set; } = true;

```

Comment:
will these additions need to be ported to the typescript generator?

Reply:
Yes.